collect_app/src/test/java/org/odk/collect/android/location/LocationTestUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.odk.collect.android.location;
+
+import android.location.Location;
+
+public class LocationTestUtils {
+    private LocationTestUtils() {
+
+    }
+
+    public static Location createLocation(String provider, double lat, double lon, double alt, float sd) {
+        Location location = new Location(provider);
+        location.setLatitude(lat);
+        location.setLongitude(lon);
+        location.setAltitude(alt);
+        location.setAccuracy(sd);
+        return location;
+    }
+
+    public static Location createLocation(String provider, double lat, double lon, double alt) {
+        Location location = new Location(provider);
+        location.setLatitude(lat);
+        location.setLongitude(lon);
+        location.setAltitude(alt);
+        return location;
```

Comment:
can't this method just return `createLocation(provider, lat, lon, alt, 0);?

Reply:
That wouldn't be the same because then the accuracy would be set. But you're right that I could call this method from the other variant and add the accuracy. Thanks, I'll do that.

Reply:
7fd6532a6